staging/src/k8s.io/apiserver/pkg/util/flowcontrol/apf_controller.go

```diff
@@ -316,28 +359,51 @@ type fsStatusUpdate struct {
 
 // digestConfigObjects is given all the API objects that configure
 // cfgCtl and writes its consequent new configState.
-func (cfgCtl *configController) digestConfigObjects(newPLs []*fctypesv1a1.PriorityLevelConfiguration, newFSs []*fctypesv1a1.FlowSchema) error {
-	fsStatusUpdates := cfgCtl.lockAndDigestConfigObjects(newPLs, newFSs)
+func (cfgCtl *configController) digestConfigObjects(newPLs []*fctypesv1a1.PriorityLevelConfiguration, newFSs []*fctypesv1a1.FlowSchema) (bool, bool, time.Duration, error) {
+	fsStatusUpdates, wait := cfgCtl.lockAndDigestConfigObjects(newPLs, newFSs)
+	var triedWrites, didWrites bool
 	var errs []error
+	fieldManager := "api-priority-and-fairness-config-consumer-v1"
```

Comment:
Looking at usage below, it's not obvious to me how this will vary between releases.  do you want to assign this based on `version.Get()` somehow?  I could get behind making this value a field on `cfgCtl`.

Reply:
We discussed this, and decided we did not need to have a fully formed plan yet, it was sufficient to put _some_ sort of version mark in here so that we have something to vary later if and when we need to.